views.py

```diff
@@ -342,10 +356,10 @@ def _cosine(mz: np.ndarray, intensity: np.ndarray, mz_other: np.ndarray,
         peak_match_idx_arr = np.asarray(peak_match_idx)[peak_match_order]
         peaks_used, peaks_used_other = set(), set()
         for peak_match_score, peak_i, peak_other_i in zip(
-                peak_match_scores_arr, peak_match_idx_arr[:, 0],
-                peak_match_idx_arr[:, 1]):
+            peak_match_scores_arr, peak_match_idx_arr[:, 0],
```

Comment:
Better to have an extra indentation here and in line 362 to have a clearer separation between the `for`/`if` condition and the code inside of those statements.